pages/index.js

```diff
@@ -0,0 +1,46 @@
+import Head from 'next/head'
+
+import { useState, useEffect, useRef } from 'react';
+import { supabase } from '../utilities/supabase';
+import { ThreeDots  } from  'react-loader-spinner'
+import { ExpressionCard } from '../components/container/index/expressioncard';
+
+import styles from '../styles/container/content.module.scss';
+import { Content } from '../sections/Content';
+
+
+function HomePage(props) {
+
+    return <>
+        <Head>
+            <title>Dicționar Urban</title>
+        </Head>
+        <Content ssrData={props.data} posts={props.HowMany}/>
+    </>
+}
+
+export async function getServerSideProps(context) {
+
+    const HowMany = 5;
+    const { data, error } = await supabase
+            .from('expressions')
+            .select()
+            .range(0, HowMany-1)
+            .order('created_at',  {ascending: false} )
+            ;
+
+    if (error || !data || data.length === 0) {
+        return {
+            notFound: true,
+        };
+    }
+
+    return {
+        props: {
+            data,
+            HowMany
```

Comment:
Ai putea pune `HowMany` în afara funcțiilor, pentru a nu o mai da ca prop.

În plus, convenția e să denumim constante cu litere mari `HOW_MANY`

Reply:
L-am pus acolo pentru că am vrut să fie ușor modificabil dintr-un singur loc, deși este folosit in mai multe componente. Care ar fi metoda ideală să fac asta?

Reply:
> Care ar fi metoda ideală să fac asta?

În afara funcțiilor

```js
const HOW_MANY = 5;
function HomePage(props) {
    // ...
}

export async function getServerSideProps(context) {
    // ...
}
```